install/src/main/resources/aspects.bzl

```diff
@@ -476,9 +476,9 @@ def _bsp_target_info_aspect_impl(target, ctx):
         for f in t.files.to_list()
     ]
 
-    java_target_info = extract_java_info(target, ctx, output_groups)
+    java_target_info = extract_java_info(target, ctx, output_groups) if not "manual" in rule_attrs.tags else None
     scala_toolchain_info = extract_scala_toolchain_info(target, ctx, output_groups)
```

Comment:
so in general lets add this if here and remove the if from the following line - i cant come with any case, but in general in this method output is created sometimes